packages/emulation/libretro-genplus/package.mk

```diff
@@ -2,25 +2,29 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libretro-genplus"
-PKG_VERSION="118e2b83873beba1a2f154a8bad8a02bdd8ea98d"
-PKG_SHA256="b6599120b736b2579cf56725b7eff8454f469c42d5039eaf6524dbb6d138eff0"
-PKG_LICENSE="Modified BSD / LGPLv2.1"
-PKG_SITE="https://github.com/libretro/Genesis-Plus-GX"
+PKG_VERSION="8f8ea7e24f7798906ebfc503ec90b478d5863d51"
+PKG_SHA256="bedc9482c394654c96aba653c8e4328ac89c127aa7536538f1efd591f3954ec7"
+PKG_LICENSE="Non-commercial"
+PKG_SITE="https://github.com/ekeeke/Genesis-Plus-GX"
```

Comment:
why do we not use https://github.com/libretro/Genesis-Plus-GX anymore?

Reply:
All these recipes have been taken from Lakka: https://github.com/libretro/Lakka-LibreELEC/blob/Lakka-v5.x/packages/lakka/libretro_cores/genesis_plus_gx/package.mk

My guess is that the libretro team has seen the value in upstreaming instead of forking everything, so I think that's the reason they're started switching to cores from other usernames.

For any Lakka package that was switched to an upstream, I've also updated the game addon in the kodi-game org to also use upstream.